bin/req_packages_anovos_py_3_7_x.sh

```diff
@@ -1,19 +1,5 @@
 sudo pip-3.7 install --upgrade cython
 sudo pip-3.7 install --upgrade pip --user
 sudo pip-3.7 install --upgrade setuptools --user
```

Comment:
@VishnuGowthem Do you know why this is necessary?

Reply:
specifically to pick up python 3.7 when 2.7 is EMR default, so this is needed. But file name could be maybe EMR version wise and